RecoJets/JetProducers/interface/PileupJetIdAlgo.h

```diff
@@ -65,9 +64,9 @@ class PileupJetIdAlgo {
 	PileupJetIdentifier internalId_;
 	variables_list_t variables_;
 
-	TMVA::Reader * reader_;
+	TMVA::Reader *reader_, *reader_jteta_0_2_, *reader_jteta_2_2p5_, *reader_jteta_2p5_3_, *reader_jteta_3_5_;
```

Comment:
since the number of these grows, please change to unique_ptr
this way the delete in the destructor are not needed